src/server/svr_jobfunc.c

```diff
@@ -1373,7 +1373,7 @@ svr_chkque(job *pjob, pbs_queue *pque, char *hostname, int mtype)
 
 	if (pque->qu_attr[QE_ATR_AclGroupEnabled].at_val.at_long)
 		if (acl_check(&pque->qu_attr[QE_ATR_AclGroup],
-			pjob->ji_wattr[(int)JOB_ATR_egroup].at_val.at_str,
+			pjob->ji_wattr[(int)JOB_ATR_euser].at_val.at_str,
```

Comment:
The new implementation looks correct. I'm not sure how it could have worked previously, as per the test case in the JIRA ticket. Could you also provide a test that demonstrates a job being rejected when the user does not have a group specified in the ACL?

I'm treating this ticket as a bug as opposed to a new feature because the existing functionality in PBS Pro is inconsistent with the way groups are used to manage things like files in Linux. In doing so, I'm hoping to avoid the extra work of creating a design document and having it reviewed.